### PR TITLE
fix(panel): remove truncating for summary and heading prop

### PR DESCRIPTION
**Pull request by y0n4**
**Related Issue:** #2823 

## Summary
Changed any truncated titles/summaries to word wrap around. For now it was in component panel. I wasn't able to find other components that had any text that gets truncated. 

Before
Screen Shot 2021-12-17 at 1 56 18 PM

After
Screen Shot 2021-12-17 at 1 55 59 PM

### CODE REVIEW

File: src/components/calcite-panel/calcite-panel.scss [hunk 1]
```diff
@@ -96,10 +96,8 @@
   .summary {
     @apply p-0
       block
-      overflow-hidden
-      whitespace-no-wrap
-      w-full;
-    text-overflow: ellipsis;
+      pointer-events-none
```
Comment (driskull) [inline src/components/calcite-panel/calcite-panel.scss:99]: why no pointer events here?

**Review by driskull — COMMENTED** [review]
Just a comment.

File: src/components/calcite-panel/calcite-panel.scss (hunk 1 above)
Comment (y0n4) [reply src/components/calcite-panel/calcite-panel.scss:99]: oh! You're right. Thanks for catching that

**Review by driskull — APPROVED** [review]
👍

**Pull request merged**